admin/src/main/resources/io/buoyant/admin/js/src/dtab_viewer.js

```diff
@@ -25,8 +29,22 @@ define(['jQuery'], function($) {
         var tuple = e.split("=>");
         return { prefix: tuple[0], dst: tuple[1] };
       });
+      this.baseDtabEdited = true;
       this.render();
       this._toggleEdit();
+      this._rerunDelegation();
+    }.bind(this));
+
+    $('#reset-dtab-btn').click(function(_e) {
+      $('.confirm-modal').modal();
+    });
+
+    $('.confirm-modal .confirm').click(function(_e) {
```

Comment:
if you're not using the passed in event object, just leave it out? `function() {`